src/main/java/org/opensearch/geospatial/action/upload/geojson/UploadGeoJSONResponse.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.action.upload.geojson;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.opensearch.action.ActionResponse;
+import org.opensearch.action.bulk.BulkItemResponse;
+import org.opensearch.action.bulk.BulkResponse;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.xcontent.ToXContentObject;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+//UploadGeoJSONResponse represents UploadGeoJSONRequest's Response
+public class UploadGeoJSONResponse extends ActionResponse implements ToXContentObject {
+    private static final String ERRORS = "errors";
+    private static final String FAILURE = "failure";
+    private static final String ID = "id";
+    private static final String MESSAGES = "messages";
+    private static final String REASON = "reason";
+    private static final String SUCCESS = "success";
+    private static final String TOOK_IN_MILLI_SECOND = "took_in_ms";
+    private static final String TOTAL = "total";
+    private static final int NO_FAILURE = 0;
+    private final BulkResponse bulkResponse;
+
+    public UploadGeoJSONResponse(BulkResponse bulkResponse) {
+        super();
+        this.bulkResponse = bulkResponse;
+    }
+
+    public UploadGeoJSONResponse(StreamInput in) throws IOException {
+        super(in);
+        this.bulkResponse = new BulkResponse(in);
+    }
+
+    @Override
+    public void writeTo(StreamOutput streamOutput) throws IOException {
+        this.bulkResponse.writeTo(streamOutput);
```

Comment:
Im not sure if it has an impact, but do we need to write super.writeTo(), before this to match the Stream ctor?

Reply:
This is from interface. So far, the implementing abstract sub class never overrides this. I don't see an impact here.